plugins/pa11y/readme.md

```diff
@@ -0,0 +1,45 @@
+# Description 
+ Pa11y is your automated accessibility testing pal. It runs accessibility tests on your pages via the command line or Node.js, so you can automate your testing process. Pa11y runs automatically on the `test:review` and `test:staging` hooks, using the information about the app from the state.
```

Comment:
**Request:** The description for the Tool Kit Plugin rather than the tool itself. We should write this description in a way that tells our consumers how this plugin will allow them to use Pa11y with their app via Tool Kit. 

```suggestion
A Tool Kit plugin to run [Pa11y](https://github.com/pa11y/pa11y). This plugin runs automatically on the `test:review` and `test:staging` hooks, using the information about the app from the state.
```

Reply:
**issue:** If #260 is merged before this PR, we will need to update the documented hooks as #260 removes the default use of the `test:staging` hook: https://github.com/Financial-Times/dotcom-tool-kit/pull/260/commits/dea7d5a61dcc7cde81e6c09fd44667b339948cbf